### PR TITLE
doc: Update imperative statement

### PR DESCRIPTION
**Pull request by addityasingh**
Update imperative statement

### CODE REVIEW

File: chapters/deprecation.adoc [hunk 1]
```diff
@@ -74,7 +74,7 @@ Clients should monitor the `Warning` header in HTTP responses to see if
 an API will be deprecated in future.
 
 [#191]
-== {MUST} Not Start Using Deprecated APIs
+== {MUST} Don't Start Using Deprecated APIs
```
Comment (whiskeysierra) [inline chapters/deprecation.adoc]: It's meant to read as one consistent sentence:

> Must not start using deprecated APIs

Comment (tkrop) [reply chapters/deprecation.adoc]: Yes. The word in the brackets is part of the sentence.

Comment (tkrop) [reply chapters/deprecation.adoc]: However, the other headings are not consistently using this pattern. This, I think, makes it harder to understand.

**Comment by tkrop** [comment]
@addityasingh thanks for the pull request. Our intention is to integrate the keyword into the rule reading this sentence as

> Must not start using deprecated APIs.

Reason is, that the imperative sentences of rules get shorter and are easier to integrate as references into other sentences.

I must admit, this is not that clear when looking on the local context of the rule. In addition, we still haven't applied this pattern to all rules. To improve this, I have created #513 and will close this issue.

**Comment by addityasingh** [comment]
Yeah. I think what was not clear for me was some of the other rules did not follow the same pattern as `Must not start using deprecated APIs.` and MUST didn't seem a part of the sentence in them. I understood them to be the norms

**Pull request closed without merging**